ThreeScalePY.py

```diff
@@ -84,31 +90,56 @@
           ]
 
 class ThreeScale:
+
+    DEFAULT_BACKEND_URI = 'https://su1.3scale.net:443'
```

Comment:
Agree, I tried "Service Management API" everywhere in the documentation, but it's a bit too long for the variable name :) Also, won't the component called "backend" in on-premise version? I'm afraid calling it `service_mgmg_api_uri` might be more confusing...